generate_data/netcdf/generate_data.sh

```diff
@@ -0,0 +1,19 @@
+# cd to this directory
+# https://stackoverflow.com/a/6393573/2700168
+cd "${0%/*}"
+
+set -e
+set -u
+set -x
+
+MVN_FLAGS=${MVN_FLAGS:-"--no-transfer-progress"}
+mvn "${MVN_FLAGS}" clean package
+
+java -cp target/jzarr-1.0.0.jar zarr_implementations.jzarr.App "$@" && {
```

Comment:
These references to "jzarr" (and really _all_ of them) will need to be changed as well.